### PR TITLE
fix: scroll behavior and css adjustments

### PR DESCRIPTION
**Pull request by janvennemann**
- Fixes scroll issues when using heading anchors
- Changes various styles to use values from the Axway style guide like padding, colors and fonts

One scroll issue that remains is that when the header anchor link is clicked, the scroll position is a little off because of the fixed navbar. Previously that was solved via negative padding/margins, but that blocked text content from being properly selected in our API docs. I opened https://github.com/vuejs/vuepress/pull/2669 for an alternative solution, but since that scroll issue is not a major blocker for now this can be merged as is.

### CODE REVIEW

File: packages/vuepress/vuepress-theme-titanium/components/PageEdit.vue [hunk 1]
```diff
@@ -138,14 +138,14 @@ export default {
   .edit-link
     display inline-block
     a
-      color lighten($textColor, 25%)
+      color #006580
```
Comment (sgtcoolguy) [inline packages/vuepress/vuepress-theme-titanium/components/PageEdit.vue:141]: I assume this came from the `primary` color in the Axway style guide? Can we assign it (and others) to known variable names and reference those?

https://zeroheight.com/3bf9d496d/p/23c5a7-design-tokens

File: packages/vuepress/vuepress-theme-titanium/styles/custom-blocks.styl [hunk 2]
```diff
@@ -8,19 +8,19 @@
     border-left-style solid
     margin 1rem 0
   &.tip
-    background-color #f3f5f7
-    border-color #19AEEF
+    background-color #EDF2F7
```
Comment (sgtcoolguy) [inline packages/vuepress/vuepress-theme-titanium/styles/custom-blocks.styl:11]: where are these colors coming from? I cannot seem to find matches in the style guide.

**Review by sgtcoolguy — COMMENTED** [review]
I think need to copy over the colors defined int he style guide "design tokens" and assign them to variables and use those. I see a handful of hex colors, only a couple of which I can trace back to some of those (It looks like the `primary` and `danger` colors?)

https://zeroheight.com/3bf9d496d/p/23c5a7-design-tokens

**Comment by sgtcoolguy** [comment]
@janvennemann I've pushed some commits to try and adhere more closely to the style guide. It adds the Axway colors to the `palette.styl` file and then tried to update hard-coded colors to refer to the named colors there. Specifically the changes are mostly around the badges (deprecated/creation only/readonly), the warning/info boxes, and inline code tags.

I also made all constants get the read only badge, and changed the h1-h4 to use the font weight/size/line height from the style guide. I'm attaching some screenshots to show most of the visual differences.

Screen Shot 2020-10-19 at 2 36 52 PM

Screen Shot 2020-10-19 at 2 37 03 PM

Screen Shot 2020-10-19 at 2 37 44 PM

Screen Shot 2020-10-19 at 2 37 10 PM

**Comment by janvennemann** [comment]
@sgtcoolguy just some notes regarding your changes.

I don't think constants need the read-only badge. They are listed under "Constants" anyway, so it should be obvious those are read-only. This made sense in our old docs since constants where just listed under "Properties", but here they are clearly separated.

I used some colors from Tailwind since the color palette in the style guide is very limited. For example, the axway teal shades available in the style guide are either too dark or too light for the inline code snippets IMHO. Same applies to the colors for badges / alert boxes. The colors from the styles guides are missing some shades to make them look good imho. For example the background color is a little to bright and is not blending as nicely with the background as before. But hopefully UI/UX team can provide us with more alternative color shades, so let's keep it this way for now.

**Pull request merged**